plone/scale/tests/test_scale.py

```diff
@@ -219,6 +216,34 @@ def testResultBuffer(self):
         self.assertEqual(result, img2)      # the return value _is_ the buffer
         self.assertEqual(result.getvalue(), img1)   # but with the same value
 
+    def testAlternativeSpellings(self):
+        """Test alternative and deprecated mode spellings and the old
+        ``direction`` arguments instead of ``mode``.
+        """
+        __import__('pdb').set_trace()
```

Comment:
Stray pdb.set_trace()

Reply:
Somehow my test fixed didn't make it to github. fixing...